projects/angular/src/accordion/stepper/models/stepper.model.ts

```diff
@@ -81,6 +87,9 @@ export class StepperModel extends AccordionModel {
   }
 
   private openFirstPanel() {
```

Comment:
Do you think it would make more sense if we rename this method to `openInitialPanel()`  and adjust the logic if needed - open initial panel, and if that is not set - open first panel?